examples/website/3d-heatmap/webpack.config.js

```diff
@@ -34,7 +34,7 @@ const CONFIG = {
 
   // Optional: Enables reading mapbox token from environment variable
   plugins: [
-    new HtmlWebpackPlugin({template: '../template.html'}),
+    new HtmlWebpackPlugin({template: 'index.html'}),
```

Comment:
Remove this and the html-webpack-plugin dependency. `index.html` can be used without any plugin.

Reply:
Remove the dependency from package.json

Reply:
@Pessimistress Updated